shared-test-resources/miniTestSuite/ValidatorTests/source/NewKeywordValidator.java

```diff
@@ -0,0 +1,9 @@
+public class NewKeywordValidator {
```

Comment:
as I understand this is not used as input. Would it increase the coverage if you have a test based on this?